run_linux.go

```diff
@@ -192,7 +192,10 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 		if err != nil {
 			return err
 		}
-		bindFiles["/etc/hosts"] = hostFile
+		// Only bind /etc/hosts if there's a network
+		if configureNetwork {
```

Comment:
Should we do the same thing for resolv.conf?

Reply:
On the fence.  Do you or others want to tip me one way or the other @rhatdan ?  I'm leaning towards yes, but not enough to tip over yet.

Reply:
IMHO /etc/hosts /etc/hostname /etc/resolve.conf should not be managed with net=none

Reply:
I agree.

Reply:
Sold!

I'll add `/etc/resolv.conf` too to the PR and will push a new variant out later today or Monday.